openmdao/docs/theory_manual/setup_stack.rst

```diff
@@ -0,0 +1,120 @@
+.. _theory_setup_stack:
+
+*********************************************************************
+The System Setup Stack: Understanding When to Use setup and configure
+*********************************************************************
+
+This document explains what happens during the OpenMDAO setup process, and how some of the model
```

Comment:
maybe "Problem setup process"?  (to be clear we are talking about the Problem setup() method, not how to install/setup OpenMDAO itself...